src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -126,6 +134,11 @@ const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
       <AnnouncementMessage message={venue?.bannerMessage} />
       {template}
       <ChatSidebar />
+      <UserProfileModal
+        userProfile={selectedUserProfile}
+        show={selectedUserProfile !== undefined}
+        onHide={closeUserProfileModal}
+      />
```

Comment:
Looks like https://github.com/sparkletown/sparkle/pull/1218#discussion_r590989276 is still outstanding? Are we planning on doing anything about it?